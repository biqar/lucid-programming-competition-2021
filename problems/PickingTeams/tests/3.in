11
? Alice Bob
+ Bob Charlie
+ Alice Charlie
? Alice Bob
? Alice Charlie
+ Bob David
? David Bob
? David Alice
+ Charlie David
? Alice David
? Bob David
