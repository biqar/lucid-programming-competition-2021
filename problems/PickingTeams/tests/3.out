no
no
yes
yes
no
yes
no
2